### PR TITLE
[gardening] Fix typos.

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

File: test/SILGen/objc_imported_generic.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ public func genericSubscriptOnAnyObject(o: AnyObject, b: Bool) -> AnyObject? {
 
 // CHECK-LABEL: sil @_TF21objc_imported_generic27genericSubscriptOnAnyObjectFT1oPs9AnyObject_1bSb_GSqPS0___
 // CHECK: bb0([[ANY:%.*]]
-// CHCEK:   [[OPENED_ANY:%.*]] = open_existential_ref [[ANY]]
+// CHECK:   [[OPENED_ANY:%.*]] = open_existential_ref [[ANY]]
```
Comment (jtbandes) [inline test/SILGen/objc_imported_generic.swift]: 😱

Comment (practicalswift) [inline test/SILGen/objc_imported_generic.swift]: @jtbandes Do you develop Swift on a Mac? If so, could you investigate the correct way to handle that? Simply changing to `CHECK` makes the Mac buildbot sad :-(

**Comment by practicalswift** [comment]
Pinging in @gottesmm regarding the `CHCEK` typo introduced in 2ccc251888907616f4022f67329468c3ea11ff1b :-)

File: test/SILGen/objc_imported_generic.swift (hunk 1 above)
Comment (jtbandes) [inline test/SILGen/objc_imported_generic.swift]: Trying, but I'm having trouble actually running the test...

**Comment by gottesmm** [comment]
@practicalswift There is probably a copy_value there from the argument before the open_existential.

**Comment by practicalswift** [comment]
@gottesmm I'm afraid I don't know enough about this to fix it. Do you know what the correct version of `CHCEK:   [[OPENED_ANY:%.*]] = open_existential_ref [[ANY]]` should be beyond the obvious `s/CHCEK/CHECK/`? :-)

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Pull request merged**